compiler/src/dotty/tools/dotc/typer/Nullables.scala

```diff
@@ -206,6 +202,25 @@ object Nullables with
       then infos
       else info :: infos
 
+  given refOps: extension (ref: TermRef) with
+
+    /* Is the use of a mutable variable out of order */
```

Comment:
Could we provide an example in the comment of what it means to use a variable "out of order"?